src/Text2Image/T2IParamInput.cs

```diff
@@ -191,7 +194,7 @@ public static (int, string) InterpretPredataForRandom(string prefix, string preD
     static T2IParamInput()
     {
         PromptTagProcessors["random"] = (data, context) =>
-        {
+        {            
```

Comment:
what editor are you using? most editors should *actively remove* things like this, not add them? 0.o

Reply:
VS2022. Weird.